bootstrap-tools.sh

```diff
@@ -40,23 +40,32 @@ done
 
 # --- END parse options
 
-UNAME=$(uname -s)
-ARCH=$(uname -m)
+UNAME="$(uname -s)"
+ARCH="$(uname -m)"
```

Comment:
is there any actuall difference in this or is just good practice to enclose in quotes?

Reply:
looks like it might not make a difference for assignment in bash; but elsewhere it avoids issues if the subshell output contains spaces etc.